MAINTAINERS.md

```diff
@@ -0,0 +1,5 @@
+## `algolia/algoliasearch-django` maintainers
+
+| Name            | Email               |
+|-----------------|---------------------|
+| Paul-Louis Nech | support@algolia.com |
```

Comment:
@julienbourdeau I believe you can now add yourself to this list 😉

Reply:
Let's say it's a bit early for that, but one day for sure 💯